frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerLifeCycle.java

```diff
@@ -141,8 +141,10 @@ private synchronized void setPort(int port) {
         private boolean error;
         private WorkerLifeCycle lifeCycle;
         private AtomicBoolean isRunning = new AtomicBoolean(true);
-        private static final org.apache.log4j.Logger loggerModelMetrics =
-                org.apache.log4j.Logger.getLogger(ConfigManager.MODEL_METRICS_LOGGER);
+        private static final Logger loggerModelMetrics =
+                LoggerFactory.getLogger(ConfigManager.MODEL_METRICS_LOGGER);
+        private static final Logger loggerModelOutput =
+                LoggerFactory.getLogger(ConfigManager.MODEL_LOGGER);
 
```

Comment:
These changes store model log in separate log files. It is much easier for user to control model log format. The only problem is that the mapping information b/w frontend worker thread id and backend worker pid is missing. It will be difficult to debug TS to detect if there is sth wrong with frontend worker thread.

Reply:
Hi, thanks for having a look. Where do you think the PID message should end up?

Reply:
Oh I see, thank you. I think that would just be a change to the log4j default settings for the MODEL_LOG. I'll experiment and probably update this PR tomorrow.

Reply:
Thanks a lot.

Reply:
I rebased and updated the default logging pattern format for MODEL_LOG to be the same as the other logs which include a thread name.

Cheers,

Alex

Reply:
It works. Thank you!